hrms/hr/doctype/shift_type/shift_type_list.js

```diff
@@ -0,0 +1,7 @@
+frappe.listview_settings["Shift Type"] = {
+	onload: function (list_view) {
+		list_view.page.add_inner_button(__("Shift Assignment Tool"), function () {
```

Comment:
Let's add this button to the Shift Assignment list view & Shift Type form view too?